week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -2,89 +2,86 @@
 // Look at the tests and see how you can fix them.
 
 function mood() {
-  let isHappy = true;
+    let isHappy = true;
 
-  if (isHappy) {
-    return "I am happy";
-  } else {
-    return "I am not happy";
-  }
+    if (!isHappy) {
```

Comment:
the fix is working but the logic doesn't stand. If I am not happy return I am happy?
a better fix is on line 5, turn it to false